src/integration-test/java/com/commercetools/sync/integration/sdk2/externalsource/inventories/utils/InventoryUpdateActionUtilsIT.java

```diff
@@ -0,0 +1,118 @@
+package com.commercetools.sync.integration.sdk2.externalsource.inventories.utils;
+
+import static com.commercetools.sync.integration.sdk2.commons.utils.ChannelITUtils.deleteChannelsFromTargetAndSource;
+import static com.commercetools.sync.integration.sdk2.commons.utils.ITUtils.deleteTypesFromTargetAndSource;
+import static com.commercetools.sync.integration.sdk2.commons.utils.InventoryITUtils.*;
+import static com.commercetools.sync.integration.sdk2.commons.utils.TestClientUtils.CTP_TARGET_CLIENT;
+import static com.commercetools.sync.sdk2.commons.utils.CustomUpdateActionUtils.buildPrimaryResourceCustomUpdateActions;
+import static com.commercetools.sync.sdk2.commons.utils.CustomValueConverter.convertCustomValueObjDataToJsonNode;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.commercetools.api.models.inventory.*;
+import com.commercetools.api.models.type.CustomFieldsDraft;
+import com.commercetools.api.models.type.CustomFieldsDraftBuilder;
+import com.commercetools.sync.sdk2.inventories.InventorySyncOptions;
+import com.commercetools.sync.sdk2.inventories.InventorySyncOptionsBuilder;
+import com.commercetools.sync.sdk2.inventories.helpers.InventoryCustomActionBuilder;
+import com.commercetools.sync.sdk2.inventories.models.InventoryEntryCustomTypeAdapter;
+import com.commercetools.sync.sdk2.inventories.models.InventoryEntryDraftCustomTypeAdapter;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import java.util.*;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class InventoryUpdateActionUtilsIT {
+
+  private static final ObjectNode CUSTOM_FIELD_VALUE =
+      JsonNodeFactory.instance.objectNode().put("en", "purple");
+
+  /**
+   * Deletes inventories and supply channels from source and target CTP projects. Populates target
+   * CTP projects with test data.
+   */
+  @BeforeEach
+  void setup() {
+    deleteInventoryEntriesFromTargetAndSource();
+    deleteTypesFromTargetAndSource();
+    deleteChannelsFromTargetAndSource();
```

Comment:
Can you try not to delete types and channels since they're not being changed in this test class anyway? It should make the tests faster and more stable.

Reply:
Also the channel is not being used in the test in this class.